### PR TITLE
Adds /duke-ds-projects route, projects-table, and download links

### LINKED ISSUE #43: Add project listing page with download links
Upon login, users are redirected to /deliveries, which shows outgoing and incoming deliveries. This page is good for managing/filtering deliveries. While users can click on a delivery to see project details (size, link, download), this is not terribly discoverable.

Add a /dds-projects/ route that lists all of the current user's projects, with links to view them in DDS and download them. This new route could be used for viewing/downloading any of the user's projects, not just the deliveries.

### PR DESCRIPTION
**Pull request by dleehr**
- Adds a /duke-ds-projects route to list all user's projects
- Includes created/updated dates in project model for sorting (provided in https://github.com/Duke-GCB/D4S2/pull/214)
- Adds index page with links to projects/deliveries
- Refactors breadcrumbs to support index page and projects navigation.

Fixes #43

### CODE REVIEW

**Comment by dleehr** [comment]
Rewrote breadcrumbs, no longer WIP

File: app/utils/breadcrumbs.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+function stripIndex(routeName) {
+  // Replace either 'index' alone or a trailing '.index
+  if (routeName) {
+    return routeName.replace(/^index$|\.index$/, '');
+  } else {
+    return null;
+  }
+
```
Comment (johnbradley) [inline app/utils/breadcrumbs.js]: Extra blank line.

File: app/templates/index.hbs [hunk 2]
```diff
@@ -1 +1,7 @@
+{{delivery-breadcrumbs}}
+<h2>Data Delivery</h2>
+<p>Welcome to Data Delivery. This application is a project of <a href="https://genome.duke.edu">Duke GCB</a>.
+  Data Delivery and is designed to facilitate transactional delivery of data sets stored in in <a href="https://dataservice.duke.edu">Duke Data Service</a>.</p>
+<p>You can use this application to list and download your {{#link-to 'duke-ds-projects'}}DukeDS Projects{{/link-to}}, or Create, Send, and Manage {{#link-to 'deliveries'}}Deliveries{{/link-to}}
```
Comment (johnbradley) [inline app/templates/index.hbs]: It wasn't immediately obvious to me that the last two links are not just informational like the previous two - they take you to where you view projects and manage deliveries.

Comment (dleehr) [reply app/templates/index.hbs]: Thanks for the feedback. This page is a first draft - something to stimulate feedback on Monday :)

**Pull request merged**